utils/client.js

```diff
@@ -2,9 +2,13 @@ import ApolloClient from 'apollo-boost';
 import { InMemoryCache } from 'apollo-cache-inmemory';
 
 const PORT = process.env.PORT || 3000;
-const path = process.env.ENV ==='development'?`http://localhost:${PORT}`:'https://vcars.herokuapp.com';
+const path = process.env.ENV;
+const basePath =
```

Comment:
Codacy has a fix for the issue: [There should be no line break before or after '='.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7128486)
```suggestion
const basePath =  path === 'development'
```